blog/info/info.py

```diff
@@ -0,0 +1,38 @@
+from flask import Blueprint
+from flask_login import login_user, current_user, logout_user, login_required
+from flask import Flask, abort, flash, Markup, redirect, flash, render_template,request, Response, session, url_for
+from datetime import datetime
+from blog.forms import ContactForm, RegistrationForm, LoginForm
+from flask_login import login_user, current_user, logout_user, login_required
+from flask_mail import Mail, Message
+import os
+
+info = Blueprint('info', __name__,
+                         template_folder='templates',
+                         static_folder='static')
+
+@info.route('/aboutme')
+def aboutMe():
+    return render_template('aboutme.html')
+
+@info.route('/contactme', methods=['GET','POST'])
+def contactMe():
+    form = ContactForm()  
+    
+    if request.method == 'POST':
+
+        if form.validate() == False:
+            return render_template('contactme.html', form=form)
+        else:
+            message = Message(sender=os.environ.get('MAIL_USERNAME'), recipients=[os.environ.get('MAIL_USERNAME')])
```

Comment:
a good candidate for the future would be to pull stuff like email into it's own class. A controller shouldn't be the place that you create emails :)